freetextresponse/mixins.py

```diff
@@ -33,7 +33,7 @@ def is_past_due(self):
         return False
 
 
-class MissingDataFetcherMixin(object):
+class MissingDataFetcher(object):  # pylint: disable=too-few-public-methods
```

Comment:
Eh, I don't know how I feel about renaming a sensible variable just because a pylint comment makes a line too long..

Reply:
What if you move the disable to above the `class` declaration instead of beside it?

It's a little more broad, but really, it could apply to any/all classes in this file; that's fine.

Reply:
And better yet, I think you can re-enable the flag again right after this class:

`# pylint: enable=too-foo-public-methods`